examples/group_level/plot_spatiotemporal_tfce.py

```diff
@@ -0,0 +1,264 @@
+    linear_model.fit(design, Y)
+
+    # --- 4) extract the resulting coefficients (i.e., betas) ---
+    # extract betas
+    coefs = get_coef(linear_model, 'coef_')
+    # only keep relevant predictor
+    betas[iteration, :] = coefs[:, pred_col]
+
+    # the matrix of coefficients has a shape of number of observations in
+    # the vertorized channel data by number of predictors;
+    # thus, we can loop through the columns i.e., the predictors)
+    # of the coefficient matrix and extract coefficients for each predictor
+    # in order to project them back to a channels x time points space.
+    lm_betas = dict()
+
+    # extract coefficients
+    beta = betas[iteration, :]
+    # back projection to channels x time points
+    beta = beta.reshape((n_channels, n_times))
+    # create evoked object containing the back projected coefficients
+    lm_betas['phase-coherence'] = EvokedArray(beta, epochs_info, tmin)
+
+    # save results
+    betas_evoked[str(subjects[iteration])] = lm_betas
+
+    # clean up
+    del linear_model
+
+###############################################################################
+# compute mean beta-coefficient for predictor phase-coherence
+
+# subject ids
+subjects = [str(subj) for subj in subjects]
+
+# extract phase-coherence betas for each subject
+phase_coherence = [betas_evoked[subj]['phase-coherence'] for subj in subjects]
+
+# average phase-coherence betas
+weights = np.repeat(1 / len(phase_coherence), len(phase_coherence))
+ga_phase_coherence = combine_evoked(phase_coherence, weights=weights)
+
+###############################################################################
+# compute bootstrap confidence interval for phase-coherence betas and t-values
+
+# set random state for replication
+random_state = 42
+random = np.random.RandomState(random_state)
+
+# number of random samples
+boot = 2000
+
+# place holders for bootstrap samples
+cluster_H0 = np.zeros(boot)
+f_H0 = np.zeros(boot)
+
+# setup connectivity
+n_tests = betas.shape[1]
+connectivity, ch_names = find_ch_connectivity(epochs_info, ch_type='eeg')
+connectivity = _setup_connectivity(connectivity, n_tests, n_times)
+
+# threshold parameters for clustering
+threshold = dict(start=.1, step=.1)
+
+# run bootstrap for regression coefficients
+for i in range(boot):
+    # extract random subjects from overall sample
+    resampled_subjects = random.choice(range(betas.shape[0]),
+                                       betas.shape[0],
+                                       replace=True)
+    # resampled betas
+    resampled_betas = betas[resampled_subjects, :]
+
+    # compute standard error of bootstrap sample
+    se = resampled_betas.std(axis=0) / np.sqrt(resampled_betas.shape[0])
+
+    # center re-sampled betas around zero
+    for subj_ind in range(resampled_betas.shape[0]):
+        resampled_betas[subj_ind, :] = resampled_betas[subj_ind, :] - \
+                                       betas.mean(axis=0)
```

Comment:
`resampled_betas -= betas.mean(axis=0)` or so should also work.

Might have to add a `[..., :]` or so to fix the broadcasting.

Reply:
nice, goof point.